files/pathnames.h.in

```diff
@@ -13,9 +13,8 @@
 #define	_PATH_NEXRC	".nexrc"
 #endif
 
-#ifndef	_PATH_PRESERVE
-#define	_PATH_PRESERVE	"@vi_cv_path_preserve@"
-#endif
+/* On linux _PATH_PRESERVE is only writable by root */
+#define	NVI_PATH_PRESERVE	"@vi_cv_path_preserve@"
```

Comment:
Is it necessary to rename this?

Reply:
Yes because otherwise it collides with _PATH_PRESERVE from paths.h which is defined to a path that is only writable by root